packages/site-demo/content/product/components/grid-column/index.mdx

```diff
@@ -0,0 +1,27 @@
+import { GridColumn } from '@lumx/react';
+import { Placeholder } from '@lumx/demo/components/content/Placeholder';
+
+# GridColumn
+
+**Display children in a customised grid, with a responsive number of column based on child min width**
+
+### Accessibility concerns
```

Comment:
This section is always at the end of component pages, just before the properties section